rest_api/v2/paths/review_booster/send_message/post.yml

```
@@ -0,0 +1,64 @@
+tags:
+  - Boosts
+summary: Send a review message
+description: >
+  This endpoint lets you send a SMS to a costumer to help him leave a message on Google My Business.
+  You need to have `WRITE` access on the business.
+  <p>Your business needs to be connected to a Google My Business location and to be subscribed to Review Booster product.
+  So this cannot be tested in Sandbox Env.
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        required:
+          - business_id
+          - target_phone
+        properties:
+          business_id:
+            $ref: '../../../components/index.yml#/schemas/BusinessId'
+          target_phone:
+            type: string
+            description: "Customer phone number, with the country calling code. Eg: +33601020304"
+          target_name:
+            type: string
+            description: Customer fullname
+            maxLength: 255
+          expeditor_name:
+            type: string
+            minLength: 3
+            maxLength: 11
+            description: >
+              It the name that will be displayed on the sms.
+              Its length should be between 3 and 11, should not contain any special character
```

```suggestion
              Its length should be between 3 and 11. It should not contain any special character
```
(or something else, bc `should` here has no subject)